### PR TITLE
Button Paddings

### PR DESCRIPTION
**Pull request by dvaldivia**
Make all the buttons resemble the UI Kit's Figma as close as possible

Screenshot 2024-05-20 at 6 03 55 PM

### CODE REVIEW

**Review by sen-io — DISMISSED** [review]
Looks great!  did you rename them in the system to match Figma?

File: src/components/ButtonGroup/ButtonGroup.tsx [hunk 1]
```diff
@@ -14,185 +14,179 @@
...
+      ),
+      "& .buttonIcon > svg": {
         color: get(
           theme,
           "buttonGroup.hoverLabelColor",
           themeColors["Color/Neutral/Text/colorTextSecondary"].lightMode,
         ),
-        borderColor: get(
+      },
+      "&.button-secondary": {
```
Comment (bexsoft) [inline src/components/ButtonGroup/ButtonGroup.tsx:137]: This is no longer called `button-secondary`, should be `button-destrictive` / `button-destructive-bare`

File: src/components/ButtonGroup/ButtonGroup.types.ts [hunk 2]
```diff
@@ -19,6 +19,5 @@ import { OverrideTheme } from "../../global/global.types";
 
 export interface ButtonGroupProps extends React.HTMLAttributes<HTMLDivElement> {
   children: React.ReactNode;
-  displayLabels?: boolean;
```
Comment (bexsoft) [inline src/components/ButtonGroup/ButtonGroup.types.ts:22]: Why are we removing this?

File: src/global/themes.ts [hunk 3]
```diff
@@ -234,6 +234,15 @@ export const lightV2 = {
...
   colorTextDisabled: "#00000040",
+  // orphan colors? (not in themeColors)
+  colorTextDestructive: "#FF3100",
+  destructiveColorBorder: "#FF3100",
+  neutralColorBorder: "#D6DAE1",
+  buttonNeutralColorTextHover: "#000000E0",
+  buttonPrimaryColorBgActive:
+    themeColors["Components/Button/Primary/colorBgActive"],
+  buttonDestructiveColorBgActive:
+    themeColors["Components/Button/Destructive/colorBgActive"],
```
Comment (bexsoft) [inline src/global/themes.ts:245]: We are deprecating `lightV2` theme colors, these can me added as colorVariant under themeColors

**Review by bexsoft — CHANGES_REQUESTED** [review]
the is an issue with DataTable 

```
[!] (plugin typescript) RollupError: [plugin typescript] src/components/DataTable/DataTable.tsx (335:11): @rollup/plugin-typescript TS2322: Type '"regular"' is not assignable to type '"text" | "neutral" | "primary" | "descructive" | "descructive-bare" | "subAction" | undefined'.
src/components/DataTable/DataTable.tsx (335:[11](https://github.com/minio/mds/actions/runs/9167178582/job/25203907856?pr=799#step:6:12))

335           variant={"regular"}

```

Can you please help us to take a look on this?

**Review by bexsoft — APPROVED** [review]

**Pull request merged**